Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1287,6 +1287,16 @@
   categories:
   - UplinkWearables
 
+- type: listing
+  id: UplinkBackpackSyndicate
+  name: uplink-backpack-syndicate-name
+  description: uplink-backpack-syndicate-desc
+  productEntity: ClothingBackpackSyndicate
+  cost:
+    Telecrystal: 2
+  categories:
+    - UplinkWearables     
```

```suggestion
  categories:
    - UplinkWearables
```

Please use an IDE so your yaml gets trimmed properly.